install(line_numbers=True, width=100, console=Console(), extra_lines=3), Failed to load pkl in current version7.0.0: Traceback (most recent call last):
  File "dynamicMatch.py", line 162, in <module>
    paraValueDict=dill.load(fr)
  File "/dataset/zhang/anaconda3/envs/rich7.0.0/lib/python3.8/site-packages/dill/_dill.py", line 287, in load
    return Unpickler(file, ignore=ignore, **kwds).load()
  File "/dataset/zhang/anaconda3/envs/rich7.0.0/lib/python3.8/site-packages/dill/_dill.py", line 442, in load
    obj = StockUnpickler.load(self)
EOFError: Ran out of input

install(line_numbers=True, width=100, console=Console(), extra_lines=3), addValueError: Traceback (most recent call last):
  File "addValueForAPI.py", line 250, in <module>
    paraValueDict=dill.load(fr)
  File "/dataset/zhang/anaconda3/envs/rich7.0.0/lib/python3.8/site-packages/dill/_dill.py", line 287, in load
    return Unpickler(file, ignore=ignore, **kwds).load()
  File "/dataset/zhang/anaconda3/envs/rich7.0.0/lib/python3.8/site-packages/dill/_dill.py", line 442, in load
    obj = StockUnpickler.load(self)
EOFError: Ran out of input


